src/lib.rs

```diff
@@ -69,11 +79,34 @@ impl Plugin for XpbdPlugin {
             .register_type::<InvInertia>()
             .register_type::<LocalCom>();
 
-        // Add stages
-        app.add_stage_before(
-            CoreStage::Update,
-            FixedUpdateStage,
-            SystemStage::parallel().with_run_criteria(run_criteria),
+        let mut xpbd_schedule = Schedule::default();
+
+        xpbd_schedule.configure_sets(
```

Comment:
I wonder if we should make these base sets instead of sets?

Reply:
Maybe? Would it primarily make system ordering less ambiguous? My knowledge of the stageless stuff is a bit shaky currently

Reply:
Yeah, that's my understanding as well. Though, I think it mostly makes sense if there is a good default base set, which I don't think there is here... Just thought I'd mention it in case anyone had an opinion.

Could always revisit it later, though.

Reply:
We can probably leave it like this for now as there's no default base set that makes sense. If there ends up being a good reason for making them base sets, someone can open a follow-up PR.